token-metadata/program/src/instruction/mod.rs

```diff
@@ -337,6 +360,8 @@ pub enum MetadataInstruction {
     #[account(2, signer, writable, name="revoke_authority", desc="Update Authority, or Delegated Authority, of Collection NFT")]
     #[account(3, name="metadata", desc="Metadata account")]
     #[account(4, name="mint", desc="Mint of Metadata")]
+    #[account(5, name="system_program", desc="System program")]
```

Comment:
Does the `metadata` account need to be writable?

Reply:
Yes, thanks!